packages/odyssey-react-mui/src/labs/FileUploadPreview.tsx

```diff
@@ -128,8 +128,13 @@ const FileUploadPreview = ({
 }: FileUploadPreviewProps) => {
   const odysseyDesignTokens = useOdysseyDesignTokens();
 
+  if (!fileNames.length) {
+    return null;
+  }
+
   return (
     <PreviewContainer
+      data-file-preview-container="true"
```

Comment:
Why are we adding a `data` prop here?

Reply:
Was needed for styling purposes [here](https://github.com/okta/odyssey/pull/2199/files#diff-ff64766022cabbb3dfec5d24e4530df06e384d834b6f5324a2f0f10b0af5cd79R1612)

Reply:
If we can pass this in, why not pass the styles as well? That way we're not depending on CSS identifying this element, we can say "we want this to be here for `margin: 0`?

Reply:
@KevinGhadyani-Okta The problem is not with styles applied to this component. The parent form control has margin on it that I want to remove when this component is rendered as a sibling to it.

Reply:
```css
[`&:has(+ [data-file-preview-container])`]: {
  marginBlockEnd: 0,
},
```
`&` is referencing `MuiFormControl` here.
This rule is accomplishing that. I'm def open to other ways of doing this but, this was the best way I could think of

Reply:
Mainly I was trying to avoid passing another prop in to `Field`

Reply:
We discussed potentially using `& + &` instead because we already had an `&:last-child` check showing our intent that this only shows for the last child.

Even though this might stylistically be a breaking change, it's because the current implementation is bugged.

The original idea is that these fields would only appear in our `Form`, component. If we don't do that, then there's a bug that's inadvertently adding bottom margin.